src/commands/instances/group/GroupRecords.ts

```diff
@@ -36,26 +58,17 @@ class GroupRecords implements Command {
         .setTitle(`${getEmoji(metric)} ${group.name} ${getMetricName(metric)} records (${period})`)
         .setDescription(this.buildList(records))
         .setURL(`https://wiseoldman.net/groups/${groupId}/records/`)
-        .setFooter(`Tip: Try ${message.prefix}group records zulrah --day`);
+        .setFooter({ text: `Tip: Try /group records metric: zulrah period: day` });
 
-      message.respond(response);
-    } catch (e) {
+      message.editReply({ embeds: [response] });
+    } catch (e: any) {
```

Comment:
if you _might_ know what type of error this is, I'd suggest `e: CommandError | unknown`
This should be type safe and avoid using `any`